Makefile

```diff
@@ -101,10 +101,6 @@ django-shell: .env .docker-build
 docs:
 	@bin/build-docs-locally.sh
 
-.PHONY: tags
```

Comment:
I like this style of declaring ``.PHONY`` along with the commands, rather than collecting all the ``.PHONY`` declarations on one line. However, it didn't prevent someone from misspelling the make target ``tag`` as ``tags``. Deleting is one way to fix it 😁

Reply:
Oops--that's probably my fault.